derived-artifacts/compiled.cmake

```diff
@@ -55,3 +55,6 @@ macro(artifact_handler CATEGORY INPUT_FILE CONFIGURATION OUTPUT TARGET_NAME)
 
 endmacro()
 register_derived_artifact("compiled" "compiled-run" "" "DIRECTORY")
+
+# Flags for Decompilation tests
+set(CFLAGS_CATEGORY_tests_analysis_Decompilation -fno-inline)
```

Comment:
Put these close to the others (see `set(CFLAGS_CATEGORY_tests_runtime...`).

Reply:
fixed